frontend/src/Frontend/Wallet.hs

```diff
@@ -270,23 +272,24 @@ getBalances model accStore = performEventAsync $ flip push accStore $ \(keys, ne
 
       -- Build a request for each chain
       requests <- flip MonoidalMap.traverseWithKey chainsToAccounts $ \chain as -> do
-        liftIO $ putStrLn $ "getBalances: Building request for get-balance on chain " <> T.unpack (_chainId chain)
+        logStrPromptly model . $(fmtLogTH LevelInfo) $ "getBalances: Building request for get-balance on chain " <> _chainId chain
```

Comment:
Is promptly the right word here? It has pretty strong connotations in the reflex space.

Reply:
It's not but it was the first thing that came to mind as I wanted to differentiate it from the `Event t (LogLevel, Text)`